### PR TITLE
Added confirmation for user canceling pr review

### PR DESCRIPTION
**Pull request by patrick-morgan**
Simple ui update to add confirmation box when user clicks to cancel a review. Works for both PRs and MRs

### CODE REVIEW

File: shared/ui/Stream/PullRequestFinishReview.tsx [hunk 1]
```diff
@@ -146,8 +147,29 @@ export const PullRequestFinishReview = (props: {
...
+											action: () => {
+												console.log("taken!");
+												cancelReview(e, pr.pendingReview?.id);
+											}
+										}
+									]
+								});
+							}}
+						>
+							Cancel fish
```
Comment (bcanzanella) [inline shared/ui/Stream/PullRequestFinishReview.tsx:172]: typo? 🐟

File: shared/ui/Stream/PullRequestFinishReview.tsx [hunk 2]
```diff
@@ -146,8 +147,29 @@ export const PullRequestFinishReview = (props: {
...
+									message: "Pending review comments will be lost.",
+									centered: true,
+									buttons: [
+										{ label: "Go Back", className: "control-button" },
+										{
+											label: "Cancel Review ",
+											className: "delete",
+											wait: true,
+											action: () => {
+												console.log("taken!");
```
Comment (bcanzanella) [inline shared/ui/Stream/PullRequestFinishReview.tsx:164]: remove console.log

**Review by bcanzanella — CHANGES_REQUESTED** [review]

File: shared/ui/Stream/PullRequestFinishReview.tsx (hunk 1 above)
Comment (patrick-morgan) [reply shared/ui/Stream/PullRequestFinishReview.tsx]: sheesh yes lol

File: shared/ui/Stream/PullRequestFinishReview.tsx (hunk 2 above)
Comment (patrick-morgan) [reply shared/ui/Stream/PullRequestFinishReview.tsx]: fixed

**Pull request merged**